source/extensions/quic_listeners/quiche/platform/quic_logging_impl.cc

```diff
@@ -11,10 +11,10 @@
 namespace quic {
 
 namespace {
-std::atomic<int>& VerbosityLogThreshold() {
-  static std::atomic<int> threshold(0);
-  return threshold;
-}
+std::atomic<int> g_verbosity_threshold;
+
+std::atomic<QuicLogSink*> g_quic_log_sink;
```

Comment:
nit: comment this global variable about when it's used and what's the common status.